webapp/mailConfig.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
this module does not only deal with mail configuration, so its name is misleading. More importantly, you have duplicated this code from the root "app" module, what we want to avoid as much as possible!

Reply:
look at the `create_user` function  in the user module, you'll see I pass the mailer as argument, thus I don't need to import those kind of module.